src/node_jsvmapi.h

```diff
@@ -203,42 +270,54 @@ NODE_EXTERN napi_status napi_make_callback(napi_env e,
 
 // Methods to work with napi_callbacks
 NODE_EXTERN napi_status napi_get_cb_args_length(napi_env e,
-                                  napi_callback_info cbinfo, int* result);
-NODE_EXTERN napi_status napi_get_cb_args(napi_env e, napi_callback_info cbinfo,
-                                  napi_value* buffer, int bufferlength);
-NODE_EXTERN napi_status napi_get_cb_this(napi_env e,
-                                  napi_callback_info cbinfo, napi_value* result);
+                                                napi_callback_info cbinfo,
+                                                int* result);
+NODE_EXTERN napi_status napi_get_cb_args(napi_env e,
+                                         napi_callback_info cbinfo,
+                                         napi_value* buffer,
+                                         int bufferlength);
+NODE_EXTERN napi_status napi_get_cb_this(
+                                napi_env e,
+                                napi_callback_info cbinfo,
+                                napi_value* result);
```

Comment:
I'll double check but I had to do that in several cases to avoid hitting the 80 char limit